### PR TITLE
chore: Add a global config to enable pprof

### PR DESCRIPTION
**Pull request by jschwinger233**
### Background

This PR adds a new global config `pprof_port` to specify (and enable/disable) pprof server. Users can also enable or disable pprof on the fly using `dae reload`.

### Checklist

- [ ] The Pull Request has been fully tested
- [ ] There's an entry in the CHANGELOGS
- [ ] There is a user-facing docs PR against https://github.com/daeuniverse/dae

### Full Changelogs

- [Implement ...]

### Issue Reference

Closes #_[issue number]_

### CODE REVIEW

**Comment by mzz2017** [comment]
我之前测试的时候发现引入 http 包会带来比较大的二进制大小膨胀，可以麻烦测试一下吗？

**Comment by jschwinger233** [comment]
@mzz2017 我对比了下 PR build: https://github.com/daeuniverse/dae/actions/runs/9441996291?pr=532 

| Arch | With pprof | Without pprof |
| :---         |     :---      |          :--- |
| arm64   | 11MB     | 10.7MB    |
| x64     | 11.6MB      | 11.3MB     |

会增加 0.3MB 大小

File: example.dae [hunk 1]
```diff
@@ -93,6 +93,9 @@ global {
     # The Client Hello ID for uTLS to imitate. This takes effect only if tls_implementation is utls.
     # See more: https://github.com/daeuniverse/dae/blob/331fa23c16/component/outbound/transport/tls/utls.go#L17
     utls_imitate: chrome_auto
+
+    # Set non-zero value to enable pprof.
+    pprof_port: 6060
```
Comment (mzz2017) [inline example.dae:98]: Recommend 0 as example value

**Review by mzz2017 — DISMISSED** [review]

File: example.dae [hunk 2]
```diff
@@ -93,6 +93,9 @@ global {
     # The Client Hello ID for uTLS to imitate. This takes effect only if tls_implementation is utls.
     # See more: https://github.com/daeuniverse/dae/blob/331fa23c16/component/outbound/transport/tls/utls.go#L17
     utls_imitate: chrome_auto
+
+    # Set non-zero value to enable pprof.
+    pprof_port: 0
```
Comment (mzz2017) [inline example.dae:98]: Insert it after line 11?

**Review by mzz2017 — APPROVED** [review]

**Comment by mzz2017** [comment]
LGTM

**Pull request merged**